app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficExporter/harLogs/converter.ts

```diff
@@ -83,10 +83,15 @@ export function createLogsHar(logs: Log[]) {
 export const convertHarJsonToRQLogs = (har: Har): Log[] => {
   const res: Log[] = har?.log?.entries?.map((entry) => {
     const requestHeaders: Record<string, string> = {};
+    const requestQueryParams: Record<string, string> = {};
     entry.request.headers.forEach((headerObj) => {
       requestHeaders[headerObj.name] = headerObj.value;
     });
 
+    entry.request.queryString.forEach((queryObject) => {
+      requestQueryParams[queryObject.name] = queryObject.value;
+    });
+
```

Comment:
Can we use the queryString map from har directly instead of converting them into key: value pairs?  Don't see any reason we need to do this specifically